src/VolleyManagement.Backend/VolleyManagement.Crosscutting.Contracts/FeatureToggles/IISDeployment.cs

```diff
@@ -5,7 +5,7 @@
     /// <summary>
     /// Describes whether we are deploying on IIS/Local environment vs Azure
     /// </summary>
-    public class IISDeployment : SimpleFeatureToggle
+    public class IisDeployment : SimpleFeatureToggle
```

Comment:
Update appsettings section of Web.Config of the UI project according to those changes/

Reply:
done 
![image](https://user-images.githubusercontent.com/17709640/37235023-0c1d65a0-2404-11e8-95b9-f198088930a5.png)